SemiconductorTestLibrary.Extensions/source/Common/HelperMethods.cs

```
@@ -17,5 +22,75 @@ internal static T GetDistinctValue<T>(IEnumerable<T> values, string errorMessage
                 throw new NISemiconductorTestException(errorMessage);
             }
         }
+
+        /// <summary>
+        /// Creates a ramp sequence of double values from outputStart to outputStop with the specified number of points.
+        /// </summary>
+        /// <param name="outputStart">The starting value of the ramp.</param>
+        /// <param name="outputStop">The ending value of the ramp.</param>
+        /// <param name="numberOfPoints">The number of points in the ramp sequence.</param>
+        /// <returns>An array of double values representing the ramp sequence.</returns>
+        /// <exception cref="ArgumentException">Thrown when numberOfPoints is less than or equal to zero.</exception>
+        /// <exception cref="ArgumentException">Thrown when outputStart is NaN or Infinity.</exception>
+        /// <exception cref="ArgumentException">Thrown when outputStop is NaN or Infinity.</exception>
+        public static double[] CreateRampSequence(double outputStart, double outputStop, int numberOfPoints)
+        {
+            if (numberOfPoints <= 0)
+            {
+                throw new ArgumentException(string.Format(CultureInfo.InvariantCulture, ResourceStrings.InvalidNumberOfPoints));
```

Not sure if we want to throw an STL exception or a generic argument exception.